check.rb

```diff
@@ -26,6 +26,7 @@
   require 'rubygems'
   # Try for Bundler version
   require 'bundler'
+  require 'bundler/vendor/uri/lib/uri'
```

Comment:
This is autoloaded by `require 'bundler'`, isn't it?

Reply:
You're correct.  Sorry, I had too many console windows open...

Sorry about the debug print.  Note to self - debug prints for when it's broken disappear once you've fixed it...

Reply:
No worries, I'll just take it out.

```suggestion
```